README.md

```diff
@@ -41,13 +29,24 @@ cd epr-re-ex-engineering-prototype
 nvm use
 ```
 
+## Start the app
+
+Start the app from the command line
+
+```bash
+npm ci --ignore-scripts
```

Comment:
I think we should add the `--ignore-scripts` flag in the `Dockerfile`, lines `12` & `34` as well. This is one of the things that came up on Friday when we were dealing with the Shai-Hulud worm protections

Reply:
Done